src/records/RecHttp.cc

```diff
@@ -98,11 +98,13 @@ mptcp_supported()
   ats_scoped_fd fd(::open("/proc/sys/net/mptcp/enabled", O_RDONLY));
```

Comment:
Do we need to try to open this regardless of the conditions?

Reply:
You're right. I will include it in the conditional, thanks!